spec/code_style_configuration_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+describe CodeStyleConfiguration do
+  it 'has a version number' do
+    expect(CodeStyleConfiguration::VERSION).not_to be nil
+  end
+
+  it 'does something useful' do
```

Comment:
remove this test and if you can think of a simple way to test the styleguide.rake task it would be great.